descriptors/ModuleDescriptor-template.json

```diff
@@ -795,6 +795,24 @@
           "delay": "1"
         }
       ]
+    },
+    {
+      "id": "subscribe-callback",
+      "version": "1.0",
```

Comment:
Do we want to go straight to 1.0 meaning this interface needs tracking with semantic versions?

Reply:
Will use 0.1 similar to what Folijet uses for inventory.